schemas/mss.xml

```diff
@@ -3874,6 +4268,21 @@ locus is used to point to values of these elements in the teiHeader.
                     </content>
                 </elementSpec>
                 
+                <!-- ADD new element "format" to describe the format of the writing substrate in prose inside "dimension". 
+                values: 
+                -	Square
+                -	Horizontal rectangle
+                -	Vertical rectangle
+                -	Ṣafīna
+                -	Irregular
+                -	Other-->
+                
+                <!--ADD new attribute "msType" to "objectDesc" to describe the Type of manuscript 
+                values:
+                -	"single": Single manuscript 
+                -	"composite" Composite manuscript
+                -	"multiple" Multiple Text Manuscript-->
```

Comment:
Don't we get this just from the structure of msParts and msItems in the document? Or is this to make machine-processing more straightforward?

Reply:
There are several suggestions in the documents I was provided to make characteristics explicit that can be computed from other elements. In the end it depends how verbose you want the schema to be.